File: Programmers/코딩테스트_입문/GwanSoon/day 7/특정 문자 제거하기.js
function solution(my_string, letter) {
  var answer = ''; 
  let count = 0; // 카운팅하는 함수 하나
  while(count < my_string.length) { // count의 숫자가 my_string의 길이보다 작을 때
      if (my_string[count] !== letter ) { // my_string 문자열의 카운트가 레터랑 다름
          answer = answer + my_string[count] // 답에다가 다시 my_string[count]
      }
      count = count + 1 // 그리고 그 뒤에 다시 + 1을 더해서 count를 n+1로 만든다.
  }
  return answer;
}

File: Programmers/코딩테스트_입문/GangHyeok/Day07/특정 문자 제거하기.java
class Solution {
    public String solution(String my_string, String letter) {
        String answer = "";
        
        for(int i =0;i<my_string.length();i++){
            if (my_string.charAt(i) != letter.charAt(0)){
                answer+=my_string.charAt(i);
            }
        }
        
        return answer;
    }
}